testsuite/integration-arquillian/tests/other/springboot-tests/pom.xml

```diff
@@ -16,11 +16,11 @@
 
         <maven.settings.file/>
         <springboot.version.option>2.4</springboot.version.option>
+        <spring.boot.adapter.version>${project.version}</spring.boot.adapter.version>
```

Comment:
Why this defaults to `${project.version}`? In the description there is we want to test older adapters.

Reply:
I would keep it as it is even if the intention is to test older versions of the adapter. There is still a springboot adapter for current releases, so it should be ok